irods_browser_app/static/js/irods.js

```diff
@@ -0,0 +1,236 @@
+$('#btn-signin-irods').on('click',function(){
+    if(sessionStorage.username)
+        $("#username").val(sessionStorage.username);
+    if(sessionStorage.password)
+        $("#password").val(sessionStorage.password);
+    if(sessionStorage.host)
+        $("#host").val(sessionStorage.host);
+    if(sessionStorage.port)
+        $("#port").val(sessionStorage.port);
+    if(sessionStorage.zone)
+        $("#zone").val(sessionStorage.zone);
```

Comment:
In the AJAX login view, I did check the credential to make sure they are valid irods credentials. If they are not valid, return a failure from the AJAX view, these session storage credentials will be set empty, so that non-valid credentials will not ever be stored in the session. In another word, values stored in the sessionStorage is guaranteed to be valid. Is this sufficient in your opinion?
I am not very sure on you second comment on prefixing or combining these. Do you mean the names of the sessionStorage variable? For example, sessionStorage.irods_username rather than sessionStorage.username? In other words, prefix all these sessionStorage variables with "irods" Thanks for any clarification on this.